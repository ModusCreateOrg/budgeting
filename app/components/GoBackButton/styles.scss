$background: #E8EDF1;
$border: #D1D2D3;

.button {
  padding: 8px 14px;
  background: $background;
  border: 1px solid $border;
  border-radius: 3px;
  color: #000;
  font-size: 0.7em;
  font-weight: bold;
  cursor: pointer;

  &:hover {
    background: darken($background, 2%);
    border-color: darken($border, 2%);
  }

  &:focus {
    outline: none;
  }
}
