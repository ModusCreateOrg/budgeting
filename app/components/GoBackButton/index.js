// @flow
import * as React from 'react';
import classnames from 'classnames';
import styles from './styles.scss';

const GoBackButton = ({ className, ...props }: any) => (
  <button {...props} className={classnames(styles.button, className)}>
    ← Go Back
  </button>
);

export default GoBackButton;
