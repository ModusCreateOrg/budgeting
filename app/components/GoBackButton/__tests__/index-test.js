import React from 'react';
import renderer from 'react-test-renderer';
import GoBackButton from '../';

it('renders correctly', () => {
  const tree = renderer.create(<GoBackButton onClick={() => null} />).toJSON();
  expect(tree).toMatchSnapshot();
});
