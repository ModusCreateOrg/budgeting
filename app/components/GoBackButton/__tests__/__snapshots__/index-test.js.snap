// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="button"
  onClick={[Function]}
>
  ‚Üê Go Back
</button>
`;
